dependencies.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+: '
+
+Copyright (C) 2020 IBM Corporation
+
+Licensed under the Apache License, Version 2.0 (the “License”);
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an “AS IS” BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+    Rafael Sene <rpsene@br.ibm.com> - Initial implementation.
+
+    This is a helper script that eases the dependency setup and 
+    configuration for the pvsadm tool. 
+'
+
+# Trap ctrl-c and call ctrl_c()
+trap ctrl_c INT
+
+function ctrl_c() {
+    echo "Bye!"
+}
+
+function identify_os() {
+
+    local OS="$(uname -s)"
+
+    case "${OS}" in
+        Linux*)     DISTRO=linux;;
+        Darwin*)    DISTRO=darwin;;
+        Catalina*)  DISTRO=darwin;;
+        *)          DISTRO="UNKNOWN:${OS}"
+    esac
+
+    ARCH=$(uname -m)
+
+    if [ "$ARCH" == "amd64" ] || [ "$ARCH" == "x86_64" ]; then
+        ARCH=amd64
+    fi
+
+    export $ARCH
+    export $DISTRO
+}
+
+
+function check_connectivity() {
+    
+    wget -q --spider http://ibm.com
+
+    if [ $? -eq 0 ]; then
+        echo "Internet connection: OK!"
+    else
+        echo "ERROR: please, check your internet connection."
+        exit
+    fi
+}
+
+
+function install_pvsadmin() {
+
+    curl -s "https://api.github.com/repos/ppc64le-cloud/pvsadm/releases" >> /tmp/pvsadmin.json
+
+    TAGS=($(cat /tmp/pvsadmin.json | jq -r '.[].tag_name'))
```

Comment:
This will return all the tags, so the subsequent commands may fail when we have more than one release published